app/views/code_schools/map.html

```diff
@@ -0,0 +1,4 @@
+<h1>Code Schools Map</h1>
```

Comment:
I honestly don't know I think it came like that, but I can have it changed to `.html.erb`.